inc/ocf_mngt.h

```diff
@@ -1015,6 +1019,47 @@ int ocf_mngt_core_set_seq_cutoff_policy_all(ocf_cache_t cache,
 int ocf_mngt_core_get_seq_cutoff_policy(ocf_core_t core,
 		ocf_seq_cutoff_policy *policy);
 
+/**
+ * @brief Set core sequential cutoff promotion request count
+ *
+ * @attention This changes only runtime state. To make changes persistent
+ *            use function ocf_mngt_cache_save().
+ *
+ * @param[in] core Core handle
+ * @param[in] thresh promotion request count
```

Comment:
Did you mean 'count' instead of 'thresh'?

Reply:
Fixed.